courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2565 @@
+{
```

Comment:
<p>just for clarify, consistency here I would change</p><p>"!pip install --user sklearn" -&gt; "!python3 -m pip install --user sklearn"</p><p> </p><p>or, alternatively replace</p><p>"!python3 -m pip freeze | grep 'tensorflow==2\|tensorflow-gpu==2' || \</p><p>!python3 -m pip install --user tensorflow==2"  with</p><p>"!pip3 install --user sklearn</p><p>!pip3 freeze | grep "tensorflow==2\|tensorflow-gpu==2" || \</p><p>pip3 install --user tensorflow==2"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>